internal/config/constants.go

```diff
@@ -1,20 +1,43 @@
 package config
 
-import "math/big"
+import (
+	"math/big"
+
+	mapset "github.com/deckarep/golang-set/v2"
+)
 
 var (
-	EthereumChainID        = big.NewInt(1)
-	GoerliChainID          = big.NewInt(5)
-	SepoliaChainID         = big.NewInt(11155111)
-	ArbitrumOneChainID     = big.NewInt(42161)
-	ArbitrumGoerliChainID  = big.NewInt(421613)
-	ArbitrumSepoliaChainID = big.NewInt(421614)
-	OptimismChainID        = big.NewInt(10)
-	OptimismGoerliChainID  = big.NewInt(420)
-	OptimismSepoliaChainID = big.NewInt(11155420)
-	BaseChainID            = big.NewInt(8453)
-	BaseGoerliChainID      = big.NewInt(84531)
-	BaseSepoliaChainID     = big.NewInt(84532)
-	LyraChainID            = big.NewInt(957)
-	LyraSepoliaChainID     = big.NewInt(902)
+	EthereumChainID         = big.NewInt(1)
+	GoerliChainID           = big.NewInt(5)
+	SepoliaChainID          = big.NewInt(11155111)
+	ArbitrumOneChainID      = big.NewInt(42161)
+	ArbitrumGoerliChainID   = big.NewInt(421613)
+	ArbitrumSepoliaChainID  = big.NewInt(421614)
+	OptimismChainID         = big.NewInt(10)
+	OptimismGoerliChainID   = big.NewInt(420)
+	OptimismSepoliaChainID  = big.NewInt(11155420)
+	BaseChainID             = big.NewInt(8453)
+	BaseGoerliChainID       = big.NewInt(84531)
+	BaseSepoliaChainID      = big.NewInt(84532)
+	LyraChainID             = big.NewInt(957)
+	LyraSepoliaChainID      = big.NewInt(902)
+	Ancient8CSepoliaChainID = big.NewInt(2863311531)
+
+	OpStack = mapset.NewSet[*big.Int](
+		OptimismChainID,
+		OptimismGoerliChainID,
+		OptimismSepoliaChainID,
+		BaseChainID,
+		BaseGoerliChainID,
+		BaseSepoliaChainID,
+		LyraChainID,
+		LyraSepoliaChainID,
+		Ancient8CSepoliaChainID,
+	)
+
+	ArbStack = mapset.NewSet[*big.Int](
```

Comment:
To reduce the diff, lets remove this until we have more use cases of networks that use the Arbitrum stack. I'm not sure if it's directly equivalent to the OP stack so we might end up modelling this differently.

Reply:
It's also out of scope for this PR